tests/test_rtc_s1_workflow.py

```diff
@@ -32,7 +32,7 @@ def _load_cfg_parameters(cfg):
     product_id = cfg.groups.product_group.product_id
     if product_id is None:
         product_id = 'OPERA_L2_RTC-S1_{burst_id}'
-    product_prefix = f'{product_id}_v{product_version}'
+    product_prefix = product_id
```

Comment:
You have changed how to define `product_prefix`, so I think few lines of the codes starting from line 26 can be removed. In specific, the code snippet below can ben removed.

```python
    product_version_float = cfg.groups.product_group.product_version
    if product_version_float is None:
        product_version = SOFTWARE_VERSION
    else:
        product_version = f'{product_version_float:.1f}'

Reply:
Good catch. Thank you, @seongsujeong